### DISCUSSION TITLE
SideKick and RaspberryPi Zero W

### DISCUSSION THREAD

**Discussion by Gilbertonl** [frntc/Sidekick64#75] [Q&A] [answered]
Should it also work with a RaspberryPi Zero W (V1.1)?
I have build two PCB's now, but it's not working. I am wondering if it should work before i go any futher.

**Comment by frntc** [comment, marked as answer]
No, the Zero W uses a different, older SoC and thus won't work. You can always check HDMI-out to see whether the RPi was booting (not with a Zero W)

**Reply by Gilbertonl** [reply]
Thanx, Then i have to wait untill Zero 2 are available again :(

**Reply by frntc** [reply]
Meanwhile you can also use a RPi 3A+/B+ with the small PCB ;-)

**Reply by Gilbertonl** [reply]
I only have a spare PI 3 (Without A+ or B+). i don't think that will work either?
If i power it up should i see the sidekick logo on the display?
